### PR TITLE
PEP8 violations

### PR DESCRIPTION
**Pull request by pheanex**
I noticed there were quite a few pep8 violations, including:
* Trailing spaces and tabs
* Depracted backticks (repr())
* Redundant parenthesis
* Unnecessary semicolons
* Inconsistent indentation
* Too many/few newlines/whitespaces
* Simplifyable assignments/comparisons
* Replaceable tests for membership with "not in/is"
* Unused imports

I don't know if you are interested in such a cleanup => Just take a look at it
FYI: I squashed all commits together for better reviewability, but if you like just some of them just let me know

### CODE REVIEW

**Comment by yarikoptic** [comment]
Hi, Thanks for taking time!  As always for such ones I am not sure since some folks (and partially me) would get then really upset with all the merge conflicts they would get for some perspective branches etc, right @sebres ?
Note also the failed tests ("is not" not "not is" ;))

**Comment by sebres** [comment]
Thx for your effort, but please read, what the law lays down: 
https://github.com/fail2ban/fail2ban/issues/876#issuecomment-122382586
Thus closed.

**Comment by sebres** [comment]
Oh, he was already itself present :)

File: config/filter.d/ignorecommands/apache-fakegooglebot [hunk 1]
```diff
@@ -6,21 +6,23 @@
 #
 import sys
 
+
 def process_args(argv):
     if len(argv) != 2:
-       sys.stderr.write("Please provide a single IP as an argument. Got: %s\n"
+        sys.stderr.write("Please provide a single IP as an argument. Got: %s\n"
```
Comment (yarikoptic) [inline config/filter.d/ignorecommands/apache-fakegooglebot]: this doesn't look right

**Comment by yarikoptic** [comment]
@sebres -- you don't have much of changes "staged" for bin/* scripts, right?  so it might actually make sense to pick those PEP8 fixes (within bin/*) + may be pruning trailing spaces within header docstrings (unlikely to conflict), and unused imports pruning (but should be done with care -- we don't have 100% code coverage yet!)

**Comment by sebres** [comment]
And what about "law"? :)

I am always an opponent of such changes (only to be PEP compliant). 
It's of course a best way to be developer of the year (or should I say commiter of ther year), but in my reality does not matter (not even a little bit).

Especially, all the guys who have a fork (or just many custom local changes) be really amazed hereafter, and it's a best way to annoy all them. 
And, imho, satisfaction is very important in the open source world.

My conclusion: let us do it really only there, where it would be needed or as your already said in new code, or lines touched in changes.

Imho, <s>paranoiacs or people who read glossy papers</s> people who really need it, can make it PEP-compliant simply with a small script or just merge together with this or similar branches.
By the way, I'm not against a PEP-compliant branch parallel to master.

**Comment by yarikoptic** [comment]
There is no "law" -- only common wisdom!  As for forks and branches -- that is why I have asked explicitly about those changes I listed -- they will not conflict with your changes (to any major extent), right?  If so we could get that done, and then whenever we move bin/* functionality under fail2ban it will come at once with better compliance with PEP8.  But it should be done before or after not during the move, so git could track the renames correctly.

An once again -- I am not treating PEP8 as a bible which we must conform, but having PEP8-cleaner code is better for long term maintenance, that is why there were those decisions to do it whenever it doesn't have "conflicts".

Making a compliant branch would not be feasible to maintain, thus wasteful.

**Comment by sebres** [comment]
Yes, but who says there are no people in the world that it will not affect?

I know an another common wisdom - don't touch a "shit", then it does not stink also (otherwise known also as Never Touch A Running System resp. NTRS). And of course I touch it also all the time, but not for such things like PEP.

You are the boss :fearful: (But I stand still to my conclusion above :beers:).

**Comment by pheanex** [comment]
So do you want some of them for just the bin/ directory?
Maybe just the not so critical ones?

**Pull request closed without merging**